django-stubs/db/models/base.pyi

```diff
@@ -49,6 +49,15 @@ class Model(metaclass=ModelBase):
     def add_to_class(cls, name: str, value: Any) -> Any: ...
     @classmethod
     def from_db(cls, db: str | None, field_names: Collection[str], values: Collection[Any]) -> Self: ...
+    def _do_update(
+        self,
+        base_qs: QuerySet[Self],
+        using: Any,
+        pk_val: Any,
+        values: Collection[Any] | None,
```

Comment:
The only caller in Django code, `Model._save_table()`, always provides list of a 3-tuple. I think it can't be `None`, unless I'm missing something.

Synthesizing based on the fact that this is passed into `UpdateQuery.add_update_fields()`

```suggestion
        values: Collection[tuple[Field, type[Model] | None, Any]],
```